docs/openapi/openapi-iam.yaml

```diff
@@ -8318,12 +8318,18 @@
                   type: object
                   additionalProperties:
```

Comment:
These openapi-iam/jwt/key.yaml are generated. You need to make your changes in

./lambda-api-graalvm/src/main/resources/cloudformation/openapi-jwt.yaml
./lambda-api-graalvm/src/main/resources/cloudformation/openapi-iam.yaml
./lambda-api-graalvm/src/main/resources/cloudformation/openapi-key.yaml

Then you can generate the files in docs/openapi by calling

./gradlew :lambda-api-graalvm:assembleOpenApiTemplate

Reply:
Once these changes are done I can generate the client-jar file so you can use it in the tests